<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="4" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-01-25T15:05:30Z" name="automationtesting" finished-at="2021-01-25T15:06:03Z" duration-ms="33155">
    <groups>
    </groups>
    <test started-at="2021-01-25T15:05:30Z" name="chrome" finished-at="2021-01-25T15:06:03Z" duration-ms="33155">
      <class name="in.automationtesting.practice.testCases.MyAccountLogin_TC007">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:in.automationtesting.practice.testCases.MyAccountLogin_TC007@4b3a45f1]" started-at="2021-01-25T15:05:30Z" name="setUp" finished-at="2021-01-25T15:05:35Z" duration-ms="5112" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="setURL()[pri:1, instance:in.automationtesting.practice.testCases.MyAccountLogin_TC007@4b3a45f1]" started-at="2021-01-25T15:05:35Z" name="setURL" description="setURL" finished-at="2021-01-25T15:05:43Z" duration-ms="7346" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setURL -->
        <test-method signature="clickOnMyAccMenu()[pri:2, instance:in.automationtesting.practice.testCases.MyAccountLogin_TC007@4b3a45f1]" started-at="2021-01-25T15:05:43Z" name="clickOnMyAccMenu" description="Click on My Account Menu" finished-at="2021-01-25T15:05:55Z" duration-ms="12268" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickOnMyAccMenu -->
        <test-method signature="Login()[pri:3, instance:in.automationtesting.practice.testCases.MyAccountLogin_TC007@4b3a45f1]" started-at="2021-01-25T15:05:55Z" name="Login" description="Click on Register" finished-at="2021-01-25T15:06:03Z" duration-ms="7560" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at in.automationtesting.practice.testCases.MyAccountLogin_TC007.Login(MyAccountLogin_TC007.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:in.automationtesting.practice.testCases.MyAccountLogin_TC007@4b3a45f1]" started-at="2021-01-25T15:06:03Z" name="tearDown" finished-at="2021-01-25T15:06:03Z" duration-ms="806" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- in.automationtesting.practice.testCases.MyAccountLogin_TC007 -->
    </test> <!-- chrome -->
  </suite> <!-- automationtesting -->
</testng-results>
